src/EventExport/Format/HTML/WebArchive/WebArchiveTemplate.php

```diff
@@ -4,14 +4,25 @@
 
 namespace CultuurNet\UDB3\EventExport\Format\HTML\WebArchive;
 
-use ValueObjects\Enum\Enum;
+use CultuurNet\UDB3\Model\ValueObject\String\Enum;
 
-/**
- * @method static WebArchiveTemplate TIPS()
- * @method static WebArchiveTemplate MAP()
- */
 final class WebArchiveTemplate extends Enum
 {
-    public const TIPS = 'tips';
-    public const MAP = 'map';
+    public static function getAllowedValues(): array
+    {
+        return [
+            'tips',
+            'map',
+        ];
+    }
+
+    public static function tips(): WebArchiveTemplate
```

Comment:
I think these methods can be added as doc blocks because the name of the method would match the value.